fork-clone-workflow.md

```diff
@@ -0,0 +1,41 @@
+# Simple fork/clone development workflow
+
+A typical fork/clone development workflow is described here. This information is intended 
+for someone new to open source development and git source code control. The fork/clone
+development workflow is not the only workflow possible. In some situations developers
+do not fork the organizational repository but rather clone the organizational repository
+directly. In this document we address only the fork/clone development workflow, not the
+workflow in which forks are not used.
+
+## Assumptions
+1. [github](git-glosasry.md#github) is the source code control system used for the project.
+1. You have aleady created a [github account](git-glossary.md#github-account). Examples here use username `joeuser`.
+1. You know the github URL for the repository on which you will be working. Examples here use https://github.com/mindfulness-at-the-computer/git-info
+1. You will at least start work on the project using the command level interface to [git](git-glossary.md#git].
```

Comment:
Closing parenthesis missing: "[git](git-glossary.md#git]"

Reply:
(Btw to find this type of problem i sometimes use the markdown preview, or when we are here in the pull request system "view" in the upper right corner)